### DISCUSSION TITLE
Messy descriptions

### DISCUSSION THREAD

**Discussion by jjpvdpoel** [home-assistant/frontend#17316] [Automations / scripts / scenes] [answered]
I'm sure this is a feature and not a bug, but I don't understand why.

![Screenshot 2023-07-15 at 18-05-43 Instellingen – Home Assistant](https://github.com/home-assistant/frontend/assets/113382981/5e9f5c94-d715-4010-9c5e-700c777d19db)

**Comment by piitaya** [comment, marked as answer]
It should be fixed in `2023.8` https://github.com/home-assistant/frontend/pull/17252

**Comment by jjpvdpoel** [comment]
Thnx
